source/common/develop/ide/editing/connect-github-codio.rst

```diff
@@ -44,6 +44,12 @@ To create a new repo, follow these steps:
    .. image:: /img/RepoURL.png
       :alt: Repo URL
 
+
+4. When disconnecting your GitHub account or repository, note that this action does not remove the associated **SSH** key from the original repository. If you wish to revoke access, you will need to remove the **SSH** key manually in the repository settings on GitHub. However, you can leave the key in place if you plan to work with the original GitHub account again in the future.
```

Comment:
This is too hidden away and not relevant to the section title Make a New repo.  Would be a lot better in its own section titled Disconnecting Codio from a Git Hub account and at the bottom of the page just above More Information